book/src/drivers.md

```diff
@@ -15,8 +15,8 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [AWS FSx for Lustre](https://github.com/aws/aws-fsx-csi-driver) | `fsx.csi.aws.com` | v0.3 | A Container Storage Interface (CSI) Driver for AWS FSx for Lustre (EBS) | Persistent | Read/Write Multiple Pods | No | No | No | ? | ?
 [Azure disk](https://github.com/andyzhangx/azuredisk-csi-driver) | `disk.csi.azure.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Azure disk | Persistent | Read/Write Single Pod | Yes | No | No | ? | ?
 [Azure file](https://github.com/andyzhangx/azurefile-csi-driver) | `file.csi.azure.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Azure file | Persistent | Read/Write Multiple Pods | Yes | No | No | ? | ?
-[CephFS](https://github.com/ceph/ceph-csi) | `cephfs.csi.ceph.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for CephFS | Persistent | Read/Write Multiple Pods | Yes | No | No | ? | ?
```

Comment:
can you also provider yes/no for the question marks?

Reply:
Sure @msau42 , followed the pattern of other drivers :)

Reply:
@msau42 Done. PTAL . Thanks!